### DISCUSSION TITLE
Is there any way to have user updatable static content without a full rebuild using SvelteKit?

### DISCUSSION THREAD

**Discussion by RickRyan26** [sveltejs/kit#3199] [Q&A] [answered]
Is there any way to have user updatable static content without a full rebuild using SvelteKit?

I assume not but would love to hear that SvelteKit will have something like ISR (Incremental Static Regeneration) or server components later this year.

Is there any hope of this feature showing up in the future?

Thank you!

**Comment by benmccann** [comment, marked as answer]
There are a couple of related issues:
- https://github.com/sveltejs/kit/issues/661
- https://github.com/sveltejs/kit/issues/2369

It's not been the highest priority for us compared to other requests

**Reply by RickRyan26** [reply]
@benmccann Thank you!!

**Reply by tonprince** [reply]
Are there any updates regarding support of ISR with Vercel adapter?

**Reply by multiplehats** [reply]
You can add ISR to routes now: https://kit.svelte.dev/docs/adapter-vercel#incremental-static-regeneration 🔥

**Reply by tonprince** [reply]
> You can add ISR to routes now: https://kit.svelte.dev/docs/adapter-vercel#incremental-static-regeneration 🔥

I tried to add the ISR conf to my `+page.server.js` file under route `/Activities/[dest]/[activity]` but when I call the route  with header `x-prerender-revalidate=XYZ`, it is not revalidated. I got a response header HIT for X-Vercel-Cache when trying to revalidate the route via Postman. What I am doing wrong? Is there anything else I have to configure properly to make this run?

**Reply by multiplehats** [reply]
> > You can add ISR to routes now: https://kit.svelte.dev/docs/adapter-vercel#incremental-static-regeneration 🔥
> 
> 
> 
> I tried to add the ISR conf to my `!page.server.js` file under route `/Activities/[dest]/[activity]` but when I call the route  with header `x-prerender-revalidate=XYZ`, it is not revalidated. I got a response header HIT for X-Vercel-Cache when trying to revalidate the route via Postman. What I am doing wrong? Is there anything else I have to configure properly to make this run?

Are you on Vercel? It's only supported for the Vercel adapter: https://github.com/sveltejs/kit/issues/661#issuecomment-1416239434

**Reply by tonprince** [reply]
Yes, I am on Vercel.

**Reply by tonprince** [reply]
I created a demo project:
https://github.com/tonprince/sveltekit-isr-demo

**Reply by gxvxc** [reply]
Stumbled on the same problem in my app too

**Comment by eecue** [comment]
Any further movement on this? Would be great to be able to do for folks not using Vercel.
